samples/go/README-ptbr.md

```diff
@@ -0,0 +1,54 @@
+# Lead Integration - Go
+
+[English version](README.md)
+
+Um simples projeto usando Go para demonstrar como receber leads do zap+.
+
+Por favor, confira nossa [documentação](https://developers.grupozap.com/) para mais informações.
+
+## Recursos Utilizados
+
+- [Go 1.16.4](https://golang.org/)
+- [Gorilla Mux 1.8.0](https://github.com/gorilla/mux)
+- [Joho GoDotEnv 1.3.0](https://github.com/joho/godotenv)
+- [Docker](https://www.docker.com/get-started)
+
+## Como instalar
+
+### Clonando
+
+``` shell
+git clone https://github.com/olxbr/crm-lead-integration.git
+cd samples/go
+```
+
+### Configurando SECRET_KEY
+
+- Modifique o valor de `SECRET_KEY` no arquivo [.env.example](.env.example)
+- Renomeie o arquivo `.env.example` para `.env`
```

Comment:
Wouldn't it be better to copy `.env.example` to `.env` and then modify the contents, GIT will show a change since the file is versioned

Reply:
Changed it on Readme